dotCMS/src/main/java/com/dotcms/scheduler/DotTask.java

```diff
@@ -0,0 +1,43 @@
+package com.dotcms.scheduler;
+
+import java.io.Serializable;
+import java.time.Duration;
+import java.util.HashMap;
+import java.util.Map;
+
+public abstract class DotTask implements Serializable {
+
+
+    private static final long serialVersionUID = 1L;
+
+    final String name() {
+        return this.getClass().getName();
+    }
+
+    public final Map<String, Serializable> map = new HashMap<>();
+
+    DotTask task;
+    String instanceId=null;
```

Comment:
Codacy found an issue: [Avoid using redundant field initializer for 'instanceId'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6006494)